packages/osd-babel-preset/package.json

```diff
@@ -9,6 +9,7 @@
   "dependencies": {
     "@babel/plugin-proposal-class-properties": "^7.16.5",
     "@babel/plugin-proposal-export-namespace-from": "^7.16.5",
+    "@babel/plugin-proposal-logical-assignment-operators": "^7.18.9",
```

Comment:
nit- should this dep be version-consistent with the rest of the `@babel/plugin-proposal-*` deps, at `^7.16.5`?

Reply:
hmm - I'm not a babel expert, but I don't see why there would be any cross-dependency between any of the plugins that should prevent us from using the latest for a new plugin.

Reply:
FWIW, I did end up changing it to `^7.16.5` in the latest commit.